src/configs/aws.rs

```diff
@@ -20,6 +35,18 @@ impl<'a> RootModuleConfig<'a> for AwsConfig<'a> {
             region: SegmentConfig::default(),
             style: Color::Yellow.bold(),
             disabled: false,
+            displayed_items: AwsItems::All,
+        }
+    }
+}
+
+impl<'a> ModuleConfig<'a> for AwsItems {
+    fn from_config(config: &Value) -> Option<AwsItems> {
+        match config {
+            Value::String(s) if s == "all" => Some(AwsItems::All),
+            Value::String(s) if s == "region" => Some(AwsItems::Region),
+            Value::String(s) if s == "profile" => Some(AwsItems::Profile),
+            _ => None,
```

Comment:
Check #383, I've put an example for parsing a enum value under the "Specification" header.

Reply:
Thanks for the heads-up it's actually a lot better.